src/upload/upload.py

```diff
@@ -50,20 +54,25 @@ def upload_process(worker, filename, save_path, data_type, data_sub_type, config
     logger.info('{}: Checking for existing MD5 from chipmunk.'.format(worker))
 
     # Logic for uploading new files based on existing and new MD5s.
-    existing_md5 = chip_data[0].get('md5Sum')
-    if existing_md5:
-        logger.info('{}: Previous MD5 found: {}'.format(worker, existing_md5))
-        if existing_md5 == generated_md5:
-            logger.info('{}: Existing MD5 matches the newly generated MD5. The file will not be uploaded.'.format(worker))
-            logger.info('{}: File: {}'.format(worker, filename))
-            logger.info('{}: Existing: {} New: {}'.format(worker, existing_md5, generated_md5))
+    if not chip_data:
+        logger.info('{}: Existing MD5 not found. A new file will be uploaded.'.format(worker))
```

Comment:
I wonder if maybe we should have a different error here? If we don't create a chip_data variable does it mean that the server isn't functioning properly? Should we stop and throw an error instead?

Reply:
first time file goes in, there is no chip data so we have to handle that case as well?

Reply:
I wasn't sure if it's an empty md5 or no chip data at all?